wavefront/data_source_alerts_test.go

```diff
@@ -0,0 +1,107 @@
+package wavefront
+
+import (
+	"testing"
+
+	"github.com/WavefrontHQ/go-wavefront-management-api/v2"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestFlattenAlert(t *testing.T) {
```

Comment:
@bwinter I know that this test is still a beast, but hard to get around it for this function.

Reply:
I think a large test or two is fine as a halfway between and integration and unit test. And per prior comments, this is all nice to think about but not necessary worth spending too much time on.